### ISSUE TITLE
[BUG] _limitString self.sub_title None

### ISSUE THREAD

**Issue by chapmanjacobd** [Sorcerio/Argparse-Interface#6]
- [Describe the bug](#describe-the-bug)
- [Expected behavior](#expected-behavior)
- [Steps to Reproduce](#steps-to-reproduce)
- [Screenshots](#screenshots)
- [Additional Context](#additional-context)

---

## Describe the bug

```
[INFO | 2024-12-29 15:40:36,980] Starting the gui.
Starting the gui.
Starting the gui.
╭───────────────────────────────────────────────────────────────────────────── Traceback (most recent call last) ──────────────────────────────────────────────────────────────────────────────╮
│ /home/xk/.local/lib/python3.13/site-packages/argui/Interface.py:138 in on_mount                                                                                                              │
│                                                                                                                                                                                              │
│   135 │   │                                                                                                                                                                                  │
│   136 │   │   # Set the title                                                                                                                                                                │
│   137 │   │   self.title = self.mainTitle                                                                                                                                                    │
│ ❱ 138 │   │   self.sub_title = self._limitString(self.mainSubtitle, 64)                                                                                                                      │
│   139 │   │                                                                                                                                                                                  │
│   140 │   │   # Install any tabs                                                                                                                                                             │
│   141 │   │   for tabsId, actions in self.__initTabsContent.items():                                                                                                                         │
│                                                                                                                                                                                              │
│ ╭───────────────────────────────────────────── locals ──────────────────────────────────────────────╮                                                                                        │
│ │ self = Interface(title='library watch', classes={'-dark-mode'}, pseudo_classes={'focus', 'dark'}) │                                                                                        │
│ ╰───────────────────────────────────────────────────────────────────────────────────────────────────╯                                                                                        │
│                                                                                                                                                                                              │
│ /home/xk/.local/lib/python3.13/site-packages/argui/Interface.py:801 in _limitString                                                                                                          │
│                                                                                                                                                                                              │
│   798 │   │   maxChars: The maximum number of characters the string should have.                                                                                                             │
│   799 │   │   postfix: The postfix to add to the string if it is longer than the limit.                                                                                                      │
│   800 │   │   """                                                                                                                                                                            │
│ ❱ 801 │   │   if len(s) <= maxChars:                                                                                                                                                         │
│   802 │   │   │   return s                                                                                                                                                                   │
│   803 │   │   return s[:maxChars - len(postfix) + 1] + postfix                                                                                                                               │
│   804                                                                                                                                                                                        │
│                                                                                                                                                                                              │
│ ╭─────────────────────────────────────────────── locals ────────────────────────────────────────────────╮                                                                                    │
│ │ maxChars = 64                                                                                         │                                                                                    │
│ │  postfix = '...'                                                                                      │                                                                                    │
│ │        s = None                                                                                       │                                                                                    │
│ │     self = Interface(title='library watch', classes={'-dark-mode'}, pseudo_classes={'focus', 'dark'}) │                                                                                    │
│ ╰───────────────────────────────────────────────────────────────────────────────────────────────────────╯                                                                                    │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
TypeError: object of type 'NoneType' has no len()
[WARNING | 2024-12-29 15:40:38,934] Unexpected return code: None
Unexpected return code: None
```

## Expected behavior

The GUI to start

## Steps to Reproduce

```
$ git clone https://github.com/chapmanjacobd/library.git
$ cb | git apply
$ ipython --pdb -m library -- wt video.db -vv --tui

:100644 100644 4099e7b8 00000000 M	library/playback/play_actions.py

diff --git a/library/playback/play_actions.py b/library/playback/play_actions.py
index 4099e7b8..66b27843 100644
--- a/library/playback/play_actions.py
+++ b/library/playback/play_actions.py
@@ -188,11 +188,15 @@ def parse_args(action, default_chromecast=None) -> argparse.Namespace:
     )
     arggroups.debug(parser)
 
     arggroups.database(parser)
     parser.add_argument("search", nargs="*")
-    args = parser.parse_intermixed_args()
+
+    from argui import Wrapper
+    wrapper = Wrapper(parser, guiFlag='--tui')
+    args = wrapper.parseArgs()
+    # args = parser.parse_intermixed_args()
     for i in range(0, 255):
         if getattr(args, f"cmd{i}") is None:
             delattr(args, f"cmd{i}")
     arggroups.args_post(args, parser)
 
```

**Comment by Sorcerio** [comment]
Fixed in [0.1.3](https://github.com/Sorcerio/Argparse-Interface/releases/tag/0.1.3). Issue was not handling missing `description` for the Parser properly.

**Comment by chapmanjacobd** [comment]
hmm... with 0.1.3 I get this:

```
[INFO | 2024-12-30 13:44:20,842] Starting the gui.
Starting the gui.
           INFO     Starting the gui.                                                                                                                                              Wrapper.py:68
╭───────────────────────────────────────────────────────────────────────────── Traceback (most recent call last) ──────────────────────────────────────────────────────────────────────────────╮
│ /home/xk/.local/lib/python3.13/site-packages/argui/Interface.py:872 in inputTypedInListChanged                                                                                               │
│                                                                                                                                                                                              │
│   869 │   │   Triggered when a typed text input *within a list* is changed.                                                                                                                  │
│   870 │   │   """                                                                                                                                                                            │
│   871 │   │   # Get the target                                                                                                                                                               │
│ ❱ 872 │   │   dest, id = event.input.name.split("_")                                                                                                                                         │
│   873 │   │                                                                                                                                                                                  │
│   874 │   │   # Get appropriate value type                                                                                                                                                   │
│   875 │   │   val = self._typedStringToValue(event.value, event.input.type)                                                                                                                  │
│                                                                                                                                                                                              │
│ ╭────────────────────────────────────────────── locals ──────────────────────────────────────────────╮                                                                                       │
│ │ event = Changed()                                                                                  │                                                                                       │
│ │  self = Interface(title='library watch', classes={'-dark-mode'}, pseudo_classes={'focus', 'dark'}) │                                                                                       │
│ ╰────────────────────────────────────────────────────────────────────────────────────────────────────╯                                                                                       │
╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
ValueError: too many values to unpack (expected 2)
[WARNING | 2024-12-30 13:44:23,052] Unexpected return code: None
Unexpected return code: None
[13:44:23] WARNING  Unexpected return code: None                                                                                                                                  Wrapper.py:103
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
```
